src/main/java/com/walmartlabs/x12/standard/StandardX12Parser.java

```diff
@@ -46,8 +50,8 @@
  * -- SE
  *
  */
-public abstract class AbstractStandardX12Parser<T extends StandardX12Document> implements X12Parser<T> {
-    private static final Logger LOGGER = LoggerFactory.getLogger(AbstractStandardX12Parser.class);
+public class StandardX12Parser<T extends StandardX12Document> implements X12Parser<T> {
```

Comment:
Should we mark this as `final` ?

Reply:
I was leaning toward making it final, I have a separate PR for that since I will have to update some of the unit tests once I make all of the protected methods private

I also have an idea for an extension point for handling unmatched transaction sets that will be in that next PR